webpack/assets/javascripts/react_app/components/Pagination/Pagination.js

```diff
@@ -2,28 +2,51 @@ import React from 'react';
 import Proptypes from 'prop-types';
 import { isEmpty } from 'lodash';
 import { Paginator } from 'patternfly-react';
-import { getURI, getURIpage, getURIperPage, changeQuery, translatePagination } from './PaginationHelper';
+import {
+  getURI,
+  getURIpage,
+  getURIperPage,
+  changeQuery,
+  translatePagination,
+} from './PaginationHelper';
 import './pagination.scss';
 
-const Pagination = ({ data }) => {
+const Pagination = (props) => {
+  const {
+    data,
+    pagination,
+    onPageSet,
+    onPerPageSelect,
+    dropdownButtonId,
+    ...otherProps
+  } = props;
+
   const urlPage = getURIpage();
   const urlPerPage = getURIperPage();
-  const className = isEmpty(data.classNames) ? 'col-md-12' : `col-md-12 ${data.classNames.pagination_classes}`;
+  const className = isEmpty(data.classNames)
+    ? 'col-md-12'
+    : `col-md-12 ${data.classNames.pagination_classes}`;
 
   return (
-      <Paginator
-        pagination={{
-          page: urlPage || 1,
-          perPage: urlPerPage || data.perPage,
-          perPageOptions: data.perPageOptions,
-        }}
-        viewType={data.viewType}
-        itemCount={data.itemCount}
-        onPageSet={page => changeQuery(getURI(), { page })}
-        onPerPageSelect={perPage => changeQuery(getURI(), { per_page: perPage })}
-        messages={translatePagination(Paginator.defaultProps.messages)}
-        className={className}
-      />
+    <Paginator
+      pagination={
+        isEmpty(pagination)
+          ? {
+              page: urlPage || 1,
+              perPage: urlPerPage || data.perPage,
+              perPageOptions: data.perPageOptions,
+            }
+          : pagination
+      }
+      viewType={data.viewType}
+      itemCount={data.itemCount}
```

Comment:
the compnent is expecting the itemCount from data, aswell as viewType but it looks like youre not inputing a viewType so its using the default one